frontend/test/pytest/test_peephole_optimizations.py

```diff
@@ -189,7 +189,7 @@ def classical_func():
             TypeError,
             match="A QNode is expected, got the classical function",
         ):
-            pipeline(classical_func)
+            pipeline({})(classical_func)
```

Comment:
Default is None so it should not be necessary no?

Reply:
Good point, removed